plugins/localwatchdog.d/20_sysinfo

```diff
@@ -43,8 +43,12 @@ function SysRq ()
     echo $ACTION > /proc/sysrq-trigger
     # Wait until logfile has changed.
     WatchFile $LOGFILE
+    if [[ $? -ne 0 ]]; then
+        logger -p local2.info "20_sysinfo - Failed to monitor journal progress" -t "restraintd"
```

Comment:
1. Log message should be consistent with `Start` and `Stop`
2. Level should be at least warning, as failure is tolerated
3. Reference to journal doesn't fit when /var/log/messages is used,
```
logger -p local2.warning -t "List of $ACTION Tasks" Change check on $LOGFILE failed
```